package/yast2-packager.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Wed Jan 27 12:16:51 UTC 2016 - lslezak@suse.cz
+
+- Fixed selecting additional products in the SLES-for-SAP product
```

Comment:
Nitpicking: the product itself is not important, the fixed/changed behavior seems to be more important than that. I might not even mention SAP here.